doc/src/mongodb.md

```diff
@@ -3,8 +3,59 @@
 # MongoDB
 
 :::{warning}
-MongoDB is currently not supported on NixOS 23.11.
-We will bring back older MongoDB versions (up to and including 4.2) for upgrading purposes only.
-As a long-term solution we are evaluating [FerretDB](https://www.ferretdb.com/) which builds on PostgreSQL
-and is compatible to MongoDB for many use cases.
+Platform support for MongoDB will be discontinued. The supported MongoDB
+versions are outdated and are only provided for the purpose of upgrading
+older machines to 23.11.
+
+New projects should not use MongoDB.
+As a replacement, we are working on [FerretDB](https://www.ferretdb.com/)
```

Comment:
We've already implemented the first beta release of a ferretdb role, consider linking to that.